apps/condo/domains/ticket/tasks/sendCreateTicketNotificationsToEmployees.js

```diff
@@ -0,0 +1,140 @@
+const { get } = require('lodash')
+const compact = require('lodash/compact')
```

Comment:
the whole lodash module is already imported on the line above, it might be worth collapsing all individual imports into the import of the entire module